pages/layout.js

```diff
@@ -17,28 +18,194 @@ export default class extends React.Component {
       ghUrl += `/tree/${committish}`;
     }
     return (
-      <div id="root">
+      <div className="root">
         <Head>
           <title>{title}</title>
           <link rel="shortcut icon" type="image/png" href={favicon} />
         </Head>
 
-        <div id="content">
+        <div className="header">
+        </div>
+
+        <div className="content">
           <Markdown escapeHtml={false} source={contents} />
         </div>
 
-        <div id="footer">
-          <a href={ghUrl}>View on <GitHub className="icon github" /></a>
+        <div className="footer">
+          <div className="wrapper">
+            <div className="repository">
+                <a className="github-link" href={ghUrl}>View on GitHub<GitHub className="icon"/></a>
+            </div>
+            <div className="copyright">
+              <a href="/">`import`</a> project by <a href="https://n8.io">@tootallnate</a>, designed by <a href="https://evilrabb.it"><EvilRabbit className="evilrabbit"/></a>
```

Comment:
Since "import project" is a noun, and "designed" is a verb, is reads kind of weird to me.

What do you think about `, design by :evilrabbit:`?